tokio-epoll-uring/src/metrics.rs

```diff
@@ -1,29 +1,69 @@
-use std::sync::atomic::{AtomicU64, Ordering};
+use std::{
+    collections::HashMap,
+    sync::{
+        atomic::{AtomicU64, Ordering},
+        RwLock,
+    },
+};
+
+use once_cell::sync::Lazy;
 
 pub struct Metrics {
     pub systems_created: u64,
     pub systems_destroyed: u64,
+    /// Slots waiters queue depth for all active systems.
+    pub slots_waiters_queue_depth: Vec<u64>,
```

Comment:
We might care about the system identifier at some point. Imagine we discover that only of the queues is full. It would be good to include them as a label.

Reply:
added in eea4d1a75b77c5cd96bad2a6b87c98d6f954f835